src/generators/python.ts

```diff
@@ -0,0 +1,258 @@
+import { Generator, GeneratorOptions } from '@fscg/generator';
+import { TypeRef, TypeSchema } from '@fscg/typeschema';
+import { pascalCase, snakeCase, sortSchemasByDeps, removeConstraints, groupedByPackage } from '@fscg/utils';
+
+// Naming conventions
+// directory naming: kebab-case
```

Comment:
Maybe better to use underscore here too ([PEP8](https://peps.python.org/pep-0008/#package-and-module-names)):

> Underscores can be used in the module name if it improves readability

Reply:
You are right. I don't know where I got this at the first time but I was sure that it should be kebab case. Fixed